juju/juju
# Add jujuc command for application credential access.

## Description of change

Add jujuc command for application  credential access. 

### Why is this change needed?

This command is needed to allow applications to access cloud credentials for CAAS features.

## QA steps

enable CAAS feature flag
```
export JUJU_DEV_FEATURE_FLAGS=caas
```

bootstrap and deploy a simple application:
```
juju bootstrap lxd qatest
````

get the cloud credentials  of the unit's model using the jujuc command
```
juju run --unit ubuntu/0 '/var/lib/juju/tools/unit-ubuntu-0/credential-get'
```

the above command should fail with the following error
```
ERROR cannot access cloud credentials: permission denied
```

"trust" the ubuntu application
```
juju trust ubuntu
```

run the command again and it should succeed displaying credential output


## Documentation changes

No

## Bug reference

No
